src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseBitbucketBambooIntegrationTest.java

```diff
@@ -113,37 +132,44 @@ public void setup() throws Exception {
         testRepo.configureRepos("testLocalRepo", "testOriginRepo");
         solutionRepo.configureRepos("solutionLocalRepo", "solutionOriginRepo");
         studentRepo.configureRepos("studentRepo", "studentOriginaRepo");
+        studentTeamRepo.configureRepos("studentTeamRepo", "studentTeamOriginaRepo");
```

Comment:
Is there a typo in `studentTeamOriginaRepo`?

Reply:
Definitely looks like a typo to me in the original `studentOriginaRepo` in the line above. I fixed both occurences.

Reply:
Perfect!